src/Vimeo/Vimeo.php

```diff
@@ -74,19 +74,21 @@ public function __construct($client_id, $client_secret, $access_token = null)
      * @param bool $json_body
```

Comment:
Just noticed this, but can you add `$headers` into the docblock?

Reply:
Good catch, added